src/content/apply/brand-designer.md

```diff
@@ -12,7 +12,7 @@ description: We're looking for brand designers to build out the Sandbox brand an
 isVisible: true
 isOpen: true
 openDate: May 18th
-closeDate: June 27th
+closeDate: July 10th
```

Comment:
Will designer app close? I thought Dajin said it won't close for the rest of the summer LOL

Reply:
he said i can update the date with the dev ones